docs/standard/exceptions/toc.yml

```diff
@@ -11,6 +11,8 @@
     href: how-to-explicitly-throw-exceptions.md
   - name: "How to: Create User-Defined Exceptions"
     href: how-to-create-user-defined-exceptions.md
+  - name: "How to: create user-defined exceptions with localized exception messages"
+    href: creating-localized-exception-messages.md
```

Comment:
I guess "ing" is not welcome in filenames, also for consistency with other articles, please consider renaming the file to `how-to-create-localized-exception-messages.md`